decidim-resource_banks/app/models/decidim/resource_bank.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module Decidim
+  # The data store for a ResourceBank in the Decidim::ResourceBank space.
+  class ResourceBank < ApplicationRecord
+    include Decidim::Participable
+    include Decidim::Publicable
+    include Decidim::HasCategory
+    include Decidim::Scopable
+    include Decidim::HasAttachments
+    include Decidim::HasAttachmentCollections
+    include Decidim::Resourceable
```

Comment:
```suggestion
    include Decidim::ParticipatorySpaceResourceable
```

I will do a PR to also change this in `Decidim::Course` model.